virttest/remote_commander/remote_master.py

```diff
@@ -259,6 +265,15 @@ def listen_errors(self, cmd):
                             remote_interface.MessengerError)):
             raise cmd
 
+    def listen_queries(self, cmd):
+        """Manage queries from slave side.
+        """
```

Comment:
Btw if you need to separate stdout/stdin from your inquires you can also use python's `multiprocessing` class with `tcp` connection. Still it'd be IMO easier to handle than using the `remoter`. Anyway I added few comments regarding naming issues and little nitpicks. If you decide like this is the way you want to go, I'll respect that.